scripts/fix/property-order.js

```diff
@@ -0,0 +1,70 @@
+/* This file is a part of @mdn/browser-compat-data
+ * See LICENSE file for more information. */
+
+'use strict';
+
+import fs from 'node:fs';
+
+import { IS_WINDOWS } from '../../test/utils.js';
+
+const propOrder = {
+  __compat: [
+    'description',
+    'mdn_url',
+    'spec_url',
+    'matches',
+    'support',
+    'status',
+  ],
+  status: ['experimental', 'standard_track', 'deprecated'],
+};
+
+function doOrder(value, order) {
+  return order.reduce((result, key) => {
+    if (key in value) result[key] = value[key];
+    return result;
+  }, {});
+}
+
+/**
+ * Return a new feature object whose first-level properties have been
+ * ordered according to Array.prototype.sort, and so will be
```

Comment:
No `sort()` call is involved here. The properties are sorted according to `propOrder`.